football_now.rb

```diff
@@ -3,6 +3,7 @@
 require 'httparty'
 require 'active_support/core_ext/string/conversions'
 require 'paint'
+# require 'byebug' only needed for debugging. Comment for no need to install it
```

Comment:
It's ok to delete it from codebase ;)

Reply:
this is why we can not have nice things 😢